core/verify_task.go

```diff
@@ -0,0 +1,119 @@
+package core
```

Comment:
recommend to integrate this file into verify_manager.go and rename verify_manager.go to trie_verify.go, one file is enough.